projects/authentication/docs/methods.md

```diff
@@ -4,31 +4,45 @@ Each publiq API supports one of three authentication methods, depending on the s
 
 ## When to use which method
 
-Which authentication method you need to use will in the first place be **determined by which API endpoint(s) you want to access**.
+Which authentication method you need to use will in the first place be **determined by which API(s) or which specific endpoint(s) you want to access**.
 
 There are 3 possible scenarios for an endpoint:
 
-1.  The endpoint requires **no authentication** at all. For example fetching the JSON representation of a specific event from UiTdatabank.
-2.  The endpoint requires **[client identification](#client-identification)**. You will only need to include your client id in the request and you don't need a token. For example Search API.
-3.  The endpoint requires a **[token](#tokens)**. You can pick whatever token type is best suited to your situation (almost every endpoint will support both types). See their respective documentation for more info.
+1.  The endpoint requires **no authentication** at all.
+2.  The endpoint requires **[client identification](#client-identification)**. You will only need to include your client id in the request either as a header or a URL parameter, and you won't need a token.
+3.  The endpoint requires a **[token](#tokens)**. If the endpoint supports both client access tokens and user access tokens you can pick whichever you prefer. See their respective documentation for more info.
 
-> An endpoint that requires authentication will require either client identification (#2) **or** a token (#3), never both.
+The following table provides a summary of the supported authentication methods on our APIs. Note that APIs may have a few exceptions on specific endpoints. For example an API may support both client access tokens and user access tokens in general, but only user access tokens on an endpoint to get the current user information (as that would be impossible without a user access token).
 
-You can find the authentication method(s) that an endpoint supports in its own documentation.
+APIs that require no authentication at all, like [UiTdatabank Taxonomy API v3](https://docs.publiq.be/docs/uitdatabank/9b63d9296d981-taxonomy-api), are not included in this table.
 
-Below you can find a short overview of how each authentication method works.
+
+API | Client identification | Client access tokens | User access tokens
+---------|----------|---------|---------
+ UiTdatabank Search API v3 | ✅ | No | No
+ [UiTdatabank Entry API v3](https://docs.publiq.be/docs/uitdatabank/5c4fcee8036e0-entry-api) | No | ✅ | ✅ 
```

Comment:
Nit: maybe we should clean up the slug and link to https://docs.publiq.be/docs/uitdatabank/entry-api?

Reply:
Good point. I suggest that we make another PR later to clean up the slugs of existing pages now that that is possible, and we then also clean up the existing links. Then we don't end up with a mix of ugly and pretty slugs. If okay with you?

Reply:
To keep things moving I'm going to merge this since you approved anyway. I can always make another PR to change this specific slug if you want, or we can do it as part of a larger cleanup a bit later.